lib/kafkaClient.js

```diff
@@ -413,6 +413,81 @@ KafkaClient.prototype.getApiVersions = function (broker, cb) {
   broker.write(request);
 };
 
+KafkaClient.prototype.getListGroups = function (callback) {
+  if (!this.ready) {
+    return callback(new Error('Client is not ready (getListGroups)'));
+  }
+  const brokers = this.brokerMetadata;
+  async.mapValuesSeries(brokers,
```

Comment:
Series is not need here since there's no no ordering requirement on the result correct? Could we change this to `mapValuesLimit` instead with a default (configurable) limit of 10.